### PR TITLE
Update docs with default serializer change

### PR DESCRIPTION
**Pull request by Peque**

### CODE REVIEW

File: docs/source/security.rst [hunk 1]
```diff
@@ -18,13 +18,16 @@ Security
...
-osBrain uses :py:mod:`pickle` module for serialization when passing messages
-between agents and can use :py:mod:`pickle` and :py:mod:`dill` for
-serialization when configuring and deploying the multi-agent architectures.
-It is well known that using pickle or dill for this purpose is a security risk.
-The main problem is that allowing a program to unpickle or undill arbitrary
-data can cause arbitrary code execution and this may wreck or compromise your
-system.
+osBrain can use different serializers when passing messages between agents and
+when configuring and deploying the multi-agent architectures. Among this
+serializers, some are considered unsafe (:py:mod:`pickle`,
```
Comment (ocaballeror) [inline docs/source/security.rst]: Only "some" are unsafe? Is there any safe serializer?

File: docs/source/security.rst [hunk 2]
```diff
@@ -18,13 +18,16 @@ Security
...
+:py:mod:`cloudpickle`...).
+
+Using these unsafe serializers is a security risk. The main problem is that
+allowing a program to deserialize arbitrary data can cause arbitrary code
+execution and this may wreck or compromise your system. Therefore, osBrain is
+meant to be run only within trusted networks (i.e.: LANs) or with properly
+encrypted/safe communications (see `Protocol encryption`_).
```
Comment (ocaballeror) [inline docs/source/security.rst]: Is the space after Protocol encryption necessary?

Comment (ocaballeror) [inline docs/source/security.rst]: Also, `serialization.rst` states that we use `pickle` by default. Is that right?

**Comment by ocaballeror** [comment]
@Peque Also, `serialization.rst` states that we use pickle by default. Is that right?

File: docs/source/security.rst (hunk 1 above)
Comment (Peque) [reply docs/source/security.rst]: Yeah, JSON, raw... Those can be used for communication between agents.

File: docs/source/security.rst (hunk 2 above)
Comment (Peque) [reply docs/source/security.rst]: Yeah, it creates a link to that section in the documentation.

**Comment by Peque** [comment]
@ocaballeror That is correct. Currently we use `pickle` by default for communication between agents. Not the most capable, but it is the fastest (and usually between agents you pass pickable data).

**Pull request merged**